aerleon/lib/cisco.py

```diff
@@ -727,8 +734,17 @@ def _FixOptions(self, proto, option):
         return sane_options
 
     def _TermletToStr(
-        self, action, proto, saddr, sport, daddr, dport, icmp_type, icmp_code, option
-    ):
+        self,
+        action: str,
+        proto: Union[int, str],
+        saddr: str,
+        sport: str,
+        daddr: str,
+        dport: str,
+        icmp_type: Union[int, str],
+        icmp_code: Union[int, str],
+        option: List[Union[str, Any]],
```

Comment:
Can we remove the Any from `option`? Also the docstring claims `option` can be None but it clearly would crash.

Reply:
Done